hack/validate/vendor

```diff
@@ -36,9 +40,9 @@ validate_vendor_diff(){
 validate_vendor_used() {
 	pkgs=$(mawk '/^[a-zA-Z0-9]/ { print $1 }' < vendor.conf)
 	for f in $pkgs; do
-	if ls -d vendor/$f  > /dev/null 2>&1; then
-		found=$(find vendor/$f -iregex '.*LICENSE.*' -or -iregex '.*COPYRIGHT.*' -or -iregex '.*COPYING.*' | wc -l)
-		if [ $found -eq 0 ]; then
+	if ls -d "vendor/$f"  > /dev/null 2>&1; then
+		found=$(find "vendor/$f" -iregex '.*LICENSE.*' -or -iregex '.*COPYRIGHT.*' -or -iregex '.*COPYING.*' | wc -l)
```

Comment:
Not for this PR, wondering if those regexes could be combined (haven't tried), i.e.

```
'.*(LICENSE|COPYRIGHT|COPYING).*'
```

(guess "NOTICE" should also be in there)

Reply:
I resisted the urge to rewrite this, yes. 

Alsop, `.*` at the beginning and the end is not needed.

Reply:
added a commit optimizing it